vhdbuilder/packer/components.json

```diff
@@ -67,7 +67,7 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-npm:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.4.45.3"
+        "v1.5.23"
```

Comment:
we usually keep the older version

and any updates on windows side here?

https://github.com/Azure/AgentBaker/blob/4b548c3ec7bb3ceafd67ff3b1c3ee07ee96e1d40/vhdbuilder/packer/generate-windows-vhd-configuration.ps1#L50

Reply:
Updated the windows versions to align with the versions in RP. I will merge this PR once i confirm theres no need to rollback.